import-scripts/fetch-dmp-data-for-import.sh

```diff
@@ -195,43 +195,43 @@ printTimeStampedDataProcessingStepMessage "MSKIMPACT data processing"
 printTimeStampedDataProcessingStepMessage "Darwin CAISIS fetch for mskimpact"
 $JAVA_BINARY $JAVA_DARWIN_FETCHER_ARGS -s mskimpact -d $MSK_IMPACT_DATA_HOME -c
 if [ $? -gt 0 ] ; then
-    cd $MSK_IMPACT_DATA_HOME ; $HG_BINARY update -C ; find . -name "*.orig" -delete
+    cd $DMP_DATA_HOME ; $GIT_BINARY git clean -fd ; $GIT_BINARY reset HEAD --hard
```

Comment:
we export "starting" files from redcap before import for all studies (impact, archer, raindance...)
However, we do "fetches" by study so if one fails earlier on, it will wipe out all the exported files and cause ALL studies to fail. 

Previous behavior was to only have one study fail (e.g archer can fail but raindance will succeed)